rootfs/rootfs/etc/rc.d/xedaemon

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+
+if [ -x /etc/init.d/xe-linux-distribution ]; then
+	/etc/init.d/xe-linux-distribution start
```

Comment:
Excellent, thanks!  Would you mind adding comments to the file pointing at those to make sure I don't ask you the same question again later? :innocent: